src/Components/Header/Header.js

```diff
@@ -1,15 +1,28 @@
 import React from "react";
-import { useHistory } from 'react-router-dom'
+import { useHistory, useParams } from 'react-router-dom'
+import logo from '../../assets/hl_logo.png'
+import single from '../../assets/single.png'
+import saved from '../../assets/saved.png'
 import './Header.css'
 
 export const Header = () => {
   const history = useHistory();
+  const { id } = useParams();
+  console.log(id)
```

Comment:
Can you remove the console.log? Also can we can use the React Link instead of manipualting the history. Is this a quick fix you could do now and re-push?

Reply:
[Fixed with this commit](https://github.com/The-Never-Ending-Story/front-end/pull/20/commits/0b7862d6e7b7acd28a00a9b751c5de50dd7600a5)